pkg/scheduler/uthelper/helper.go

```diff
@@ -144,6 +144,9 @@ func (test *TestCommonStruct) CheckAll(caseIndex int) (err error) {
 
 // CheckBind check expected bind result
 func (test *TestCommonStruct) CheckBind(caseIndex int) error {
+	if test.BindsNum != len(test.Bind) {
```

Comment:
And what's the difference between this and following validation `if len(test.Bind) != len(binder.Binds)`.

Reply:
test.BindsNum is the expected binding number of pods ,
test.Bind is the expected detail binding info indicate which pod binded to which node,
binder.Binds is the actual got detail binding info indicate which pod is bind to which node.